src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -98,10 +151,17 @@ public static Vector4 Darken(Vector4 backdrop, Vector4 source)
     /// <param name="source">The source vector.</param>
     /// <returns>The <see cref="Vector4"/>.</returns>
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public static Vector4 Lighten(Vector4 backdrop, Vector4 source)
-    {
-        return Vector4.Max(backdrop, source);
-    }
+    public static Vector4 Lighten(Vector4 backdrop, Vector4 source) => Vector4.Max(backdrop, source);
+
+    /// <summary>
+    /// Returns the result of the "Lighten" compositing equation.
+    /// </summary>
+    /// <param name="backdrop">The backdrop vector.</param>
+    /// <param name="source">The source vector.</param>
+    /// <returns>The <see cref="Vector256{Single}"/>.</returns>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static Vector256<float> Lighten(Vector256<float> backdrop, Vector256<float> source)
+        => Avx.Max(backdrop, source);
```

Comment:
I don't know how to do `OverlayValueFunction` below.

Reply:
I'm a little rusty but something like this should work:

```cs
private static readonly Vector256<float> Half = Vector256.Create(0.5f);
private static readonly Vector256<float> One = Vector256.Create(0.5f);
private static readonly Vector256<float> Two = Vector256.Create(0.5f);

[MethodImpl(MethodImplOptions.AggressiveInlining)]
public static Vector256<float> OverlayValueFunction(Vector256<float> backdrop, Vector256<float> source)
{
    var left = Avx.Multiply(Avx.Multiply(Two, backdrop), source);
    var right = Avx.Subtract(One, Avx.Multiply(Avx.Multiply(Two, Avx.Subtract(One, source)), Avx.Subtract(One, backdrop)));

    var cmp = Avx.CompareGreaterThan(backdrop, Half);
    return Avx.BlendVariable(left, right, cmp);
}
```

SIMD implementation must calculate both cases but it's still a bit faster than scalar:

```
BenchmarkDotNet=v0.13.4, OS=Windows 11 (10.0.22621.1265)
Unknown processor
.NET SDK=7.0.100
  [Host]     : .NET 6.0.12 (6.0.1222.56807), X64 RyuJIT AVX2
  DefaultJob : .NET 6.0.12 (6.0.1222.56807), X64 RyuJIT AVX2


| Method |     Mean |    Error |   StdDev |
|------- |---------:|---------:|---------:|
| Scalar | 92.27 ns | 0.313 ns | 0.277 ns |
|    Avx | 35.93 ns | 0.356 ns | 0.333 ns |
```

Reply:
Ah that's awesome. Thanks! That lets me complete the first objective.